sdk/helper/compressutil/compress.go

```diff
@@ -107,10 +93,6 @@ func Compress(data []byte, config *CompressionConfig) ([]byte, error) {
 		buf.Write([]byte{CompressionCanarySnappy})
 		writer = snappy.NewBufferedWriter(&buf)
 
-	case CompressionTypeLZ4:
-		buf.Write([]byte{CompressionCanaryLZ4})
-		writer = lz4.NewWriter(&buf)
-
 	default:
 		return nil, errors.New("unsupported compression type")
```

Comment:
```suggestion
		return nil, fmt.Errorf("unsupported compression type: %v", config.Type)
```

perhaps, and likewise with decoding, keeping the constants? I agree, that way anyone who happens to have weird entries lying around can very clearly see why.